config/examples/Creality/Ender-3/BigTreeTech SKR Mini E3 3.0/Configuration.h

```diff
@@ -1173,7 +1172,7 @@
 
 // Enable this feature if all enabled endstop pins are interrupt-capable.
 // This will remove the need to poll the interrupt pins, saving many CPU cycles.
-#define ENDSTOP_INTERRUPTS_FEATURE
+//#define ENDSTOP_INTERRUPTS_FEATURE
```

Comment:
If this can be enabled, it's usually helpful. AFAIK all ARM board endstop pins should be interrupt-capable.

Reply:
> If this can be enabled, it's usually helpful. AFAIK all ARM board endstop pins should be interrupt-capable.

See https://github.com/MarlinFirmware/Configurations/issues/823

Reply:
This could be due to an uncaught conflict between interrupt-enabled pins and related timers or ISRs. It would be worth an investigation by checking the STM32 datasheets to see if the problematic endstop pins coincide with timers or ISRs and whether we can change how we initialize the related pins, timers, and ISRs.